packages/core-js/modules/esnext.data-view.set-float16.js

```diff
@@ -3,9 +3,45 @@ var $ = require('../internals/export');
 var uncurryThis = require('../internals/function-uncurry-this');
 var aDataView = require('../internals/a-data-view');
 var toIndex = require('../internals/to-index');
-var packIEEE754 = require('../internals/ieee754').pack;
 var f16round = require('../internals/math-f16round');
 
+var EPSILON = 2.220446049250313e-16; // Number.EPSILON
+var INVERSE_EPSILON = 1 / EPSILON;
+
+var roundTiesToEven = function (n) {
+  return n + INVERSE_EPSILON - INVERSE_EPSILON;
+};
+
+var minInfinity16 = 65520; // (2 - 2 ** -11) * 2 ** 15
+var minNormal16 = 0.000061005353927612305; // (1 - 2 ** -11) * 2 ** -14
+var recMinSubnormal16 = 16777216; // 2 ** 10 * 2 ** 14
+var recSignificandDenom16 = 1024; // 2 ** 10;
+
+function packFloat16(value) {
```

Comment:
Use named functions only for exported features (an issue of some minification cases), here use `var packFloat16 = ...`.